README.md

```diff
@@ -57,6 +57,21 @@ A diagram of this running 5 steps on a single sample image may help explain how
 ## Installation
 
 This library conveniently works in the browser, in Node, and on the command line (CLI).
+To install the app, run the following commands in a command line:
+
+* `git clone https://github.com/vaibhavmatta/image-sequencer`
+* `cd image-sequencer`
+* `npm install`
+
+## Running locally:
+
+To run the app locally on your machine, run the following commands in a command line:
```

Comment:
I believe you forgot to mention the command here.

Reply:
Command is mentioned just after line 68.

Reply:
yeah my mistake